src/App/store/Store.js

```diff
@@ -132,6 +132,25 @@ class Store {
   @action.bound
   getBackground() {
     if (localStorage.background) {
+      var background = JSON.parse(localStorage.background);
+      // check to see if we set the file attribute
+      if (typeof background["file"] !== "undefined") {
```

Comment:
Nit: be consistent with you string decalrations. Change this to `'undefined'`, i.e. use single quotes.